### ISSUE TITLE
wrangled with sidebar icons

### ISSUE THREAD

**Issue by TheSamsa** [onivim/oni2#3642]
Hello,
my sidebar icons are wrangled up. A day before the SVG changes I grabbed the new master branch version and they appeared like this:
![2021-05-31-090724_75x186_scrot](https://user-images.githubusercontent.com/13667299/120436097-ca2a5680-c37e-11eb-9e43-122bffa8b4f6.png)

And today I fetched the newest changes WITH the SVG commit, but they still look the same. Something is off, but I can't grasp whats wrong?

I am on a Manjaro Linux with i3wm and use the Hack Nerd Font for the editor. Also I tried switching themes, it did not work

Here is the log output
[oni2.log](https://github.com/onivim/oni2/files/6582544/oni2.log)

**Comment by TheSamsa** [comment]
After digging a bit into the issue, I noticed if I remove the float values inside FeatherIcons.re the icons are displayed (nearly) propperly.
I then tried to use a , instead of a . for the values and it also worked.
This hints to my locale settings of my linux machine, I ran oni with `export LC_NUMERIC=en_GB.UTF-8; Oni2` and it worked.

This means this does interfere with locale settings on someones unix machine. (I have german numericals and time but english global language)

But I don't know how revery/oni could handle this correctly. I might fix it if someone can hint me to a solution
